components/Dialogs/examples/DialogsAlertCustomizationExampleViewController.swift

```diff
@@ -20,42 +20,7 @@ import MaterialComponents.MaterialCollections
 import MaterialComponents.MaterialDialogs_DialogThemer
 import MaterialComponents.MaterialTypographyScheme
 
-class CustomAlertViewController: UIViewController {
```

Comment:
Was this removed because it wasn't be used?

Reply:
yes. it wasn't used anywhere, and there was no point in just renaming it...